components/RefinementList.js

```diff
@@ -43,6 +45,7 @@ class RefinementList extends React.Component {
     var rootClass = cx(this.props.cssClasses.root);
     var listClass = cx(this.props.cssClasses.list);
     var itemClass = cx(this.props.cssClasses.item);
+    var transformData = this.props.transformData || identity;
 
```

Comment:
There is no performance problem involve here, and it makes the code easier to grasp (no if). IMHO using the identity makes sense here.